Frontend/src/components/mentee-profile/MenteeProfile.jsx

```diff
@@ -1,13 +1,125 @@
-import { useState} from 'react'
+import {useContext, useEffect, useState} from 'react'
+import { UserContext } from '../../UserContext.jsx';
 import { useNavigate } from 'react-router-dom';
 import './MenteeProfile.css'
+import MentorProfileModal from '../mentor-profile/MentorProfileModal.jsx';
+import config from '../../../config.js';
 
 const PLACEHOLDER = "https://ralfvanveen.com/wp-content/uploads/2021/06/Placeholder-_-Glossary.svg";
-
+const skillsList = [
+    'Communication',
+    'Teamwork',
+    'Problem-Solving',
+    'Time Management',
+    'Leadership',
+    'Adaptability',
+    'Critical Thinking',
+    'Project Management',
+    'Creativity',
+    'Technical Writing',
+    'Customer Service',
+    'Public Speaking',
+    'Data Analysis',
+    'Strategic Planning',
+    'Negotiation',
+    'Conflict Resolution',
+    'Financial Management',
+    'Marketing',
+    'Sales',
+    'Research',
+    'Programming',
+    'Networking',
+    'Graphic Design',
+    'Content Creation',
+    'SEO',
+    'Social Media Management',
+    'UX/UI Design',
+    'Cybersecurity',
+    'Cloud Computing',
+    'Database Management',
+    'Machine Learning',
+    'Artificial Intelligence',
+    'Supply Chain Management',
+    'Quality Assurance',
+    'Product Development',
+    'Business Analysis',
+    'Human Resources',
+    'Training and Development',
+    'Legal Knowledge',
+    'Foreign Languages'
+];
```

Comment:
this would be good to use a string mapped enum for, so that you dont have to deal with errors from things getting mispelled

Reply:
Here's one example to use enums: https://www.geeksforgeeks.org/enums-in-javascript/

Reply:
Resolved